File: day2/src/solution_tests.exs
ExUnit.start()

defmodule MainTests do
  use ExUnit.Case, async: true

  test "position_multiplied" do
    input = ['forward 5', 'down 5', 'forward 8', 'up 3', 'down 8', 'forward 2']
    assert Part1.position_multiplied(input) == 150
    assert Part2.position_multiplied(input) == 900
  end
end

File: day3/src/solution_tests.jl
using Test
include("solution.jl")

input = [
  0 0 1 0 0;
  1 1 1 1 0;
  1 0 1 1 0;
  1 0 1 1 1;
  1 0 1 0 1;
  0 1 1 1 1;
  0 0 1 1 1;
  1 1 1 0 0;
  1 0 0 0 0;
  1 1 0 0 1;
  0 0 0 1 0;
  0 1 0 1 0;
]

@test power_consumption(input) == 198
@test life_support_rating(input) == 230
